test/WebSites/BasicWebSite/Startup.cs

```diff
@@ -28,6 +28,7 @@ public void ConfigureServices(IServiceCollection services)
                     // Filter that records a value in HttpContext.Items
                     options.Filters.Add(new TraceResourceFilter());
                 })
+                .SetCompatibilityVersion(CompatibilityVersion.Latest)
```

Comment:
:ok: Searching for `IClassFixture<WebApplicationFactory<`, I see this case is unique. Everything else is using the `MvcTestFixture` and that sets the `CompatibilityVersion`.

Separately, we set the `CompatibilityVersion` in the fixture to avoid having to change every web site. We're also able to run some scenarios with both `Version2_0` and `Version2_1`.